pypiserver/_app.py

```diff
@@ -53,15 +58,8 @@ def protector(*args, **kwargs):
 @app.hook("before_request")
 def log_request():
     log.info(config.log_req_frmt, request.environ)
-
-
-@app.hook("before_request")
-def print_request():
-    parsed = urlparse(request.urlparts.scheme + "://" + request.urlparts.netloc)
-    request.custom_host = parsed.netloc
-    request.custom_fullpath = (
-        parsed.path.rstrip("/") + "/" + request.fullpath.lstrip("/")
-    )
+    request.custom_host = urlparse(
+            f"{request.urlparts.scheme}://{request.urlparts.netloc}")
```

Comment:
I see here we used to have `request.custom_host = parsed.netlock` but I actually don't see any references to `.custom_host` anywhere in the code. Maybe this could also go 🤔 .

Reply:
![image](https://github.com/user-attachments/assets/a477bd4b-11ce-4408-94cc-b2101839e132)